### PR TITLE
Fix Pandas 0.2x breaking Google BigQuery change

### PR DESCRIPTION
**Pull request by NielsZeilemaker**
Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [X] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/1179) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-1179

### Description
- [X] Here are some details about my PR, including screenshots of any UI changes:
Added pandas-gbq to setup.py and change the import

### Tests
- [X] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
Only import path has changed, so I didn't add unit-tests

### Commits
- [X] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

**Comment by criccomini** [comment]
To keep packaging a bit simpler, can you include the pandas_gbq as part of gcp_api, rather than its own subpackage?

File: setup.py [hunk 1]
```diff
@@ -251,6 +252,7 @@ def do_setup():
             'all_dbs': all_dbs,
             'async': async,
             'azure': azure,
+            'bigquery': bigquery,
```
Comment (criccomini) [inline setup.py]: I think this can go, too, right?

**Comment by criccomini** [comment]
Can you make sure the CI passes, too? Might require a rebase.

**Comment by NielsZeilemaker** [comment]
@criccomini I did a rebase, so the CI should pass again.

**Comment by NielsZeilemaker** [comment]
@criccomini I did a rebase, so the CI should pass again.

**Comment by bolkedebruin** [comment]
@criccomini @NielsZeilemaker the travis' builds on master are current failing with errors in BigQuery/Pandas can you guys have look?

**Pull request closed without merging**